### PR TITLE
fix import statement

### PR DESCRIPTION
**Pull request by themousepotato**
Correct import statement and update according to https://godoc.org/github.com/google/go-github/github.

### CODE REVIEW

**Comment by themousepotato** [comment]
I signed it!

**Comment by themousepotato** [comment]
Hey @gmlewis !

I tested the code. It wasn't working. I updated it as https://godoc.org/github.com/google/go-github/github. If this isn't correct, it should be corrected. From my testing experience, I can say that this is correct. Please test it once in the meantime.

**Comment by gmlewis** [comment]
Thanks, @themousepotato.

What version of Go are you using?

Is your environment variable `GO111MODULE` set?
If so, what is its value?

Have you read the docs on Go modules and specifically release information here:
https://github.com/golang/go/wiki/Modules#releasing-modules-v2-or-higher
?

One thing we are trying to do is move toward supporting `Go v1.11` and beyond by adding support for Go modules.

**Comment by themousepotato** [comment]
@gmlewis Sorry for the late reply. Here you go:
`go version go1.11 linux/amd64`

`GO111MODULE` env var is not set. I only tried to keep in sync with the docs on https://godoc.org/github.com/google/go-github/github. Thanks!

**Comment by gmlewis** [comment]
Thanks, @themousepotato, for the extra info.
@marwan-at-work - do you have time to take a look at this PR and comment please?

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ If you're interested in using the [GraphQL API v4][], the recommended library is
 ## Usage ##
 
 ```go
-import "github.com/google/go-github/v18/github"
+import "github.com/google/go-github/github"
```
Comment (marwan-at-work) [inline README.md]: we should update the docs to say this: 

```
Usage: 

with go modules enabled (GO111MODULE=on or outside GOPATH): import "github.com/google/go-github/v18/github"	

with go modules disable: import "github.com/google/go-github/github"
```

Then also mention that v18 and above is go1.9.x and above. 
For anything below, people can use v17.

**Pull request merged**